source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -5459,6 +5459,7 @@ Octopus.Client.Model
   {
     .ctor()
     Boolean CanPasswordBeEdited { get; set; }
+    DateTimeOffset Created { get; set; }

```

Comment:
@andrewabest Should this be `protected set;` be be consistent with [this comment](https://github.com/OctopusDeploy/OctopusDeploy/pull/12437#discussion_r898505299)?

Reply:
No, not in the Octopus.Clients repository.